src/main/java/duke/DukeDateFormatter.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DukeDateFormatter {
+    // DateTimeFormatter is immutable and thread-safe, and thus the recommended
+    // approach is to store it in a static constant.
+    private static final DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+    private static final DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
```

Comment:
For static final (constants), you might want to write it in uppercase with underscores (eg. INPUT_FORMATTER)?